po/glossary/gnc-glossary.txt

```diff
@@ -76,6 +76,7 @@
 "credit note"	"A document that you give to a client that says you owe money to the client, i.e. the opposite of an invoice"
 "currency"	"The system of money used in a country"
 "Custom"	"Custom print format (i.e. according to the user's wishes) as opposed to a template choice."
+"Dashboard"	"-"
```

Comment:
Without an  explanation it does not make much sense.

Reply:
@fellen: Done.

Reply:
I should have searched it before: https://en.wikipedia.org/wiki/Dashboard_(disambiguation)
Is everything important included?

Reply:
@fellen: Is my suggestion so bad?

Reply:
I don't know it. I didn't know the word and have not used the report. So I posted just an idea and will trust you.